### PR TITLE
refactor/improve codebase

### PR DESCRIPTION
**Pull request by gustavovnicius**
# Description 

- Bump outdated packages
- Remove mocha/expect
- Setup Jest

### CODE REVIEW

File: dist/actions.js [hunk 1]
```diff
@@ -1 +1,61 @@
...
\ No newline at end of file
+"use strict";
```
Comment (p10ns11y) [inline dist/actions.js]: Just curious why build files are formatted. Is it because `eslint --fix` included dist folder?

Comment (gustavovnicius) [reply dist/actions.js]: New babel version

Comment (gustavovnicius) [reply dist/actions.js]: I added a flag and proper `NODE_ENV` var

File: test/component.immutable.reducer.spec.js [hunk 2]
```diff
@@ -1,120 +1,162 @@
...
+
+import I18n from '../immutable';
+import { i18nState } from '../immutable';
+import { setLanguage, setTranslations } from '../dist/actions';
+
+import TransWithoutParams from './components/TransWithoutParams';
+
+describe('translations in reducer (immutable)', () => {
+  it('set all translations object', () => {
+    const store = createStore(
```
Comment (p10ns11y) [inline test/component.immutable.reducer.spec.js]: we can move store outside and make it reusable
```
const getStore = () => createStore(
    combineReducers({ i18nState }),
    applyMiddleware(thunk),
);
```

**Pull request merged**